docs/source/glossary/glossary.md

```diff
@@ -41,17 +41,23 @@ The data manifest is a file that includes the locations of all fields of view fo
 A collection of Image Tiles corresponding to a specific volume or plane of the sample, under which the signal for all channels and all imaging rounds were acquired. All tiles within this FOV are the same size, but the manifest allows for different spatial coordinates for different imaging rounds or channels (to accommodate slight movement between rounds, for example).
 In microscopy, a field of view corresponds to the area of the camera sensor, and many such fields of view are expected to be taken per tissue slice.
 
-### Indices (Field of View):
-Indices are used to navigate within a field of view, and comprise the physical dimensions of pixel space (X, Y), the z-plane, channel (C) and imaging round (R) of a field of view.
-
 ### Image Tile
 A single plane, single channel 2D image. In the manifest, each tile has information about it’s (X,Y,Z) coordinates in space, and information about which imaging round (R) and/or fluorescence channel (C) it was acquired under.
 
 ### Coordinates (Tile):
 Coordinates refer to the physical location of a Tile with respect to some independent reference.
 
-### Indices (Tile):
-Indices indicate which round, channel, and z-plane a given (x, y) Tile corresponds to.
+### Axes (Tile):
+Each tile is located along three axes, which are round, channel, and z-plane.  Each (x, y) Tile is assigned values for each of these three axes.
+
+### Labels (Tile):
```

Comment:
I'd reorganize the last sentence: 

```diff
- The possible set of values for a given axis is the labels for that axis.
+ Labels are the possible set of values for each axis.
```

Reply:
I'd also not repeat the Axis definition under Labels (I had to compare the sentences word by word to see if there was new info) Not sure if "Because it is possible to crop by round, channel, or z-plane, not all values are valid." best belongs under Axes or Labels, though.

Reply:
Done, and done.